src/sentry/models/user.py

```diff
@@ -182,6 +182,11 @@ def get_verified_emails(self):
     def has_unverified_emails(self):
         return self.get_unverified_emails().exists()
 
+    def has_usable_password(self):
+        if self.password == "" or self.password is None:
```

Comment:
Probably worth documenting why we're adding this here

Reply:
Oh huh, swear I had a comment about it. I'll add it in again.